### PR TITLE
Fix missing Dockerfile in CI

### PR DESCRIPTION
**Pull request by kurtismullins**
In the Python3 work, we have introduced multiple changes to CI. Specifically, we have moved all test-dependencies from the base Dockerfile(s) into Dockerfile.cirun.

There is now the following error:
```
Unable to find image 'quay-ci:943072a-687-ci-run' locally

docker: Error response from daemon: pull access denied for quay-ci, repository does not exist or may require 'docker login'.
```

**Jira:**
https://issues.redhat.com/browse/PROJQUAY-218

### CODE REVIEW

**Comment by kurtismullins** [comment]
This does not fix _every_ test as can be seen in this PR's Travis CI build. However, it's a good start and I want to cleanup how CI is performed overall.

File: scripts/ci [hunk 1]
```diff
@@ -68,10 +75,11 @@ clean_cache() {
...
 
 fail_clean() {
-    if [[ $TRAVIS_TEST_RESULT -ne 0 ]]; then
-        echo "Job failed. Cleaning cache..."
-        clean_cache
-    fi
+    #if [[ $TRAVIS_TEST_RESULT -ne 0 ]]; then
+    #    echo "Job failed. Cleaning cache..."
+    #    clean_cache
+    #fi
```
Comment (thomasmckay) [inline scripts/ci]: Just remove this?

File: scripts/ci [hunk 2]
```diff
@@ -204,7 +212,7 @@ postgres() {
 case "$1" in
     lint)
         run_black
-        # run_flake8  TODO: Run this after offending files have been fixed. Ensure appropriate Python version is used.
+        #run_flake8  # TODO: Enable this once we're ready to tackle Flake8 revealed errors
```
Comment (thomasmckay) [inline scripts/ci]: I thought we were going to enable this on python3 branch?

Comment (kurtismullins) [reply scripts/ci]: Not until we're ready to fix the errors it reveals (non-trivial)

File: scripts/ci (hunk 1 above)
Comment (kurtismullins) [reply scripts/ci]: I'll clean it up in a future revision :) I have another approach I want to take on CI in general that is much quicker and simpler.

**Pull request merged**